vtex/faststore
# feat: Add priceCurrency to Offer

## What's the purpose of this pull request?
We were not returning the right value on priceCurrency. This caused some issues as seen on this thread: https://vtex.slack.com/archives/C0164SHMAV9/p1656518534885009

## How it works?
I added a loader that grabs the same info from the current sales channel from the context. This loader then, uses the sales channel api to fetch the right value for price currency

## How to test it?
Open the site in a pdp, grab a product's ld+json and paste it on Google's rich result tester. Make sure this field is ok

### Starters Deploy Preview
- https://github.com/vtex-sites/gatsby.store/pull/139
- https://github.com/vtex-sites/nextjs.store/pull/161
